app.py

```diff
@@ -63,36 +63,30 @@ async def on_voice_state_update(member, before, after):
                 await before.channel.edit(name="General")  # Reset channel name
 
 
+session_roles = {782273117321166888, 782423045578948618, 782423808271056917, 853135207862370324, 853135509488009216, 853135836597583902, 931662314074169364, 921725505051459664, 921725841677901905, 698285991110246441}
+subject_roles = {688354722251276308, 688355303808303170, 871702273640787988, 685837416443281493, 685837450895032336, 685837475939221770, 667769546475700235, 688357525984509984, 685837363003523097, 685836740774330378, 677411157434171404, 688357284920819717, 668927421512155142, 688359807203278986, 685837255138607182, 688356986798211074, 688362197096988688, 688356500506148884, 685836973004423169, 685837280396705803, 688360519735967766, 688362250951852085, 685837171491471401, 685836834005319721, 868056692942835742, 932660913054580766, 688361654852780048, 871587296330260570, 685836866166980661, 688361164161548361, 688361596149563431, 685836889990627335, 871587728737849344, 886884445657890826, 853588511252545566, 853587095317643264, 853588416774930432, 853586114380300298, 853586292435451904, 853586450141413416, 883190243623333888, 883190176644468758, 881366711645921310, 868323020769493034, 883190289211228170, 868324666102661140, 698291687981580308}
+
 @bot.event
 async def on_raw_reaction_add(reaction):
     guild = bot.get_guild(GUILD_ID)
     user = await guild.fetch_member(reaction.user_id)
     if user.bot:
         return
     is_rr = rrDB.get_rr(str(reaction.emoji), reaction.message_id)
+    verified = discord.utils.get(guild.roles, name="Verified")
```

Comment:
Use the utility function `getRole` over here

Reply:
Done.